references/detection/coco_eval.py

```diff
@@ -246,7 +245,7 @@ def loadRes(self, resFile):
 
     # print('Loading and preparing results...')
     # tic = time.time()
-    if isinstance(resFile, torch._six.string_classes):
+    if isinstance(resFile, str):
```

Comment:
Hum, this is a change in behavior from the previous implementation, which considered both `str` and `bytes`.

I wonder if in this case it wouldn't be better to stick with `torch._six.string_classes`?

Reply:
Sure, I will revert it. 

---

Do you know why we include `bytes` for [python3](https://github.com/pytorch/pytorch/blob/e3daf7018499fbcb52de4018543fd9a0306ccca6/torch/_six.py#L46-L49)? For python2 `torch._six.string_classes` is simply `basestring`, which according to the [`six` documentation](https://six.readthedocs.io/index.html#six.string_types) is just `str` in python3.

Reply:
Great question, and I'm afraid it might be because we were dealing with output of a few IO functions in Python might be `str` or `bytes`, depending on how you opened the file.

Can you open an issue in torchvision so that we keep this in mind? We might want to revisit this in the future, and remove `bytes` support altogether

Reply:
See #2038